config/default.go

```diff
@@ -110,6 +110,9 @@ MaxTxLifetime = "3h"
 		ByteGasCost = 16
 		MarginFactor = 1
 		Enabled = false
+	[Sequencer.StreamServer]
```

Comment:
We can add an "Enabled" parameter to specify if the StreamServer must be initiated or not. Instead to check if Port and Filename are set. I think in this way is more clear when the StreamServer is enabled or not

Reply:
I finally added the enabled config param